src/app/shared/layouts/explore-layout/explore-layout.component.ts

```diff
@@ -1,15 +1,61 @@
 import { Component, OnInit } from '@angular/core';
+import { FormControl, FormGroup } from '@angular/forms';
+import { getRatingStringValue } from '../../models/filter/rating';
+import { FillFilterService } from '../../services/fill-filter.service';
 
 @Component({
   selector: 'app-explore-layout',
   templateUrl: './explore-layout.component.html',
   styleUrls: ['./explore-layout.component.scss']
 })
 export class ExploreLayoutComponent implements OnInit {
+  panelOpenState = false
+  filterForm: FormGroup
+  selectedInputs={
```

Comment:
Rework this approach using FormControl API: https://angular.io/api/forms/FormControl#usage-notes